XIVSlothCombo/Combos/RDM.cs

```diff
@@ -891,17 +1117,18 @@ const int
 
                     return RDM.Verflare;
                 }
-                if (dualcastBuff || accelBuff || swiftcastBuff || HasEffect(RDM.Buffs.Chainspell) || level <= 18)
+                if (dualcastBuff || accelBuff || swiftcastBuff || HasEffect(RDM.Buffs.Chainspell) )
                     return OriginalHook(RDM.Impact);
 
-                if (level <= 18)
-                    return RDM.Verthunder2;
+                if (level < RDM.Levels.Verthunder2)
```

Comment:
I saw a level map somewhere by the way, we should implement that solution-wide.

Reply:
Thought the levels maps where by job files and not centralized, at least the ones I looked at.

Reply:
Yeah I mean the AST changes. Maybe do it when you get around to it